core/src/cluster_info.rs

```diff
@@ -1175,7 +1174,9 @@ impl ClusterInfo {
         blobs.iter().for_each(|blob| {
             let blob = blob.read().unwrap();
             let from_addr = blob.meta.addr();
-            deserialize(&blob.data[..blob.meta.size])
+            bincode::config()
+                .limit(PACKET_DATA_SIZE as u64)
+                .deserialize(&blob.data[..blob.meta.size])
```

Comment:
Hm gossip blobs are limited to BLOB_SIZE (defined in packet.rs).maybe this should match that limit instead of packet data size?

Maybe we can reduce blob size to packet size in a later PR.